tracex_project/extraction/logic/orchestrator.py

```diff
@@ -129,19 +128,24 @@ def run(self, view=None):
                 patient_journey=self.get_configuration().patient_journey
             )
         patient_journey = ". ".join(patient_journey_sentences)
+        self.configuration.update(patient_journey=patient_journey)
```

Comment:
I think this line should be removed. The way it is now, the original patient journey wont be displayed in the result view, correct? We would use the preprocess one. I think the changes made by the preprocessor should not be displayed to the user.